docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,156 @@
+| [**PlatformVM : Import Avax from X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildImportTx-XChain.ts) | _X Chain >> P Chain_ |
+| [**PlatformVM : Export Avax to X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildExportTx-XChain.ts) | _P Chain >> X Chain_ |
+| [**AVM : Import Avax from P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildImportTx-PChain.ts) | _P Chain >> X Chain_ |
+| [**EVM : Export Avax to P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-pchain.ts) | _C Chain >> P Chain_ |
+| [**PlatformVM : Import Avax from C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildImportTx-CChain.ts) | _C Chain >> P Chain_ |
+| [**PlatformVm : Export Avax to C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildExportTx-CChain.ts) | _P Chain >> C Chain_ |
+| [**EVM : Import Avax from P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-PChain.ts) | _P Chain >> C Chain_ |
+
+
+### Step 2 - Modify your export
+
+:::info
+
+The script in this tutorial uses **X Chain <-> C Chain** transfers. Depending on which chains you want to interact with, the scripts you use could be different. Please check the table above for more information.
+
+:::
+
+Select the **examples/avm** folder to view the AvalancheJS X-Chain examples. To send AVAX from the X-Chain to the C-Chain, select [```avm/buildExportTx-cchain-avax.ts```](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts)
+
+![Image for post](/img/ajs-export-cchain-res.png)
+
+#### Modify your Avalanche network configuration
+
+```js
+const ip: string = "localhost" | "custom"
+```
+```js
+const port: number = 9650 | custom
+```
+```js
+const protocol: string = "http" | "https"
+```
+
+```js
+const networkID: number =
+  0: "custom"
+  1: "avax"
+  2: "cascade"
+  3: "denali"
+  4: "everest"
+  5: "fuji"
+  1337: "custom"
+  12345: "local"
+  ```
+
+Example:
+```js
+const ip: string = "localhost"
+const port: number = 13445
+const protocol: string = "http"
+const networkID: number = 1337
+```
+
+#### Edit the amount of AVAX you want to send:
+By default the script sends the wallet's entire AVAX balance:
+
+```js
+const balance: BN = new BN(getBalanceResponse.balance)
+const amount: BN = balance.sub(fee)
+```
+
+Change the amount by creating a new _BN_ variable: (```value```) and assigning it a string value (```"10000000000000000"```) 
+```js
+  const value: BN = new BN("10000000000000000")
+  const amount: BN = value.sub(fee)
+  ```
+
+### Step 3 - Run the Export and Import scripts
+
+```zsh
+avalanchejs $ ts-node examples/avm/buildExportTx-cchain-avax.ts
+Success! TXID: Rgg2412kaczRYC3taasvG6bYoqG7tBQG6WfacNdumKDKsVWpF
+avalanchejs $ ts-node examples/evm/buildImportTx-xchain.ts               
```

Comment:
`examples/evm/buildImportTx-xchain.ts` was not mentioned before. Please explain what it does.

Reply:
https://github.com/ava-labs/avalanche-docs/blob/ebc232b5fa0f117a69328a08c3a7142613b6c13a/docs/quickstart/cross-chain-transfers.md?plain=1#L237